client/src/js/components/App.jsx

```diff
@@ -27,13 +29,22 @@ class App extends React.Component {
           {id: 18,name: "Cheese Cake", github: "toCakeOrToTart", absencePoints: 12, health: 86},
         ]}
     };
+    this.selectStudent = this.selectStudent.bind(this);
+  }
+
+  selectStudent(studentId) {
+    let selectedStudent = this.state.cohort.students.filter(x => x.id === studentId)[0];
```

Comment:
Can you always use parens around parameters?  This is legitimate ES6 syntax but my preference is that we not omit the parameters when using ES6 anonymous functions.